.github/workflows/dependabot.yml

```diff
@@ -0,0 +1,62 @@
+name: Test Dependabot PRs
+
+on:
+  pull_request:
+
+env:
+  GU_SUPPORT_WORKERS_LOAD_S3_CONFIG: false
+
+jobs:
+  run_node_tests:
+    if: github.actor == 'dependabot[bot]'
+    name: test dependabot
+    runs-on: ubuntu-latest
+    steps:
+    # ---- Logging ---- #
+      - name: Env
+        run: env
+
+      - name: Dump GitHub context
+        id: github_context_step
+        run: echo $JSON
+        env:
+          JSON: ${{ toJSON(github) }}
+
+    # ---- Setup ---- #
+      - name: Checkout repo
+        uses: actions/checkout@v2
+
+      - name: Setup Node - support-frontend
+        uses: guardian/actions-setup-node@v2.4.1
+        with:
+          cache: 'yarn'
+          cache-dependency-path: support-frontend/yarn.lock
+
+      - name: Install - support-frontend
+        run: yarn
+        working-directory: support-frontend
+
+      - name: Setup Node - CDK
```

Comment:
Have you tried it without this step? It may not need it

Reply:
No I just copied it over from the support-frontend-build workflow - I'll have a go at dropping it now

Reply:
Yeah we don't need it - I'll create a ticket to go in and completely unify the versions of node used